src/components/App/App.css

```diff
@@ -19,6 +19,32 @@
 .body {
   padding: 50px;
   flex-grow: 1;
+  margin: 0 auto;
+  max-width: 480px;
+}
+
+@media screen and (min-width: 820px) {
+  .body {
+    max-width: 720px;
+  }
+}
+
+@media screen and (min-width: 1060px) {
+  .body {
+    max-width: 960px;
+  }
+}
+
+@media screen and (min-width: 1300px) {
+  .body {
+    max-width: 1200px;
+  }
+}
+
+@media screen and (min-width: 1540px) {
+  .body {
+    max-width: 1440px;
```

Comment:
I think we only need 1 max width, and it should be larger than this. I don't know what the exact value should be, but there is no reason we need to limit screen real estate artificially. I think the point of max width is so we don't have to care about yuge screens being really hard to read.